lib/utils/normalize.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * A set of utilities to coerce arguments.
+ */
+
+
+/**
+ * @param {Branch | String}
+ * @return {String} The branch fullname
+ */
+function branchName(branch) {
+    return typeof branch === 'string'
+        ? branch
+        : branch.getFullName();
+}
+
+module.exports = {
+    branchName: branchName
```

Comment:
I would only name it `branch` so that you can do `normalize.branch()` but up to you.